### ISSUE TITLE
Block Spotify ads

### ISSUE THREAD

**Issue by bakunin1848** [DNSCrypt/dnscrypt-proxy#1462]
https://raw.githubusercontent.com/CHEF-KOCH/Spotify-Ad-free/master/filters/Spotify-HOSTS.txt

It seems that this link is no longer valid.

If someone wants to confirm it and remove it from the blacklist.

Greetings

![bl](https://user-images.githubusercontent.com/11910873/91647083-bf8d0a80-ea56-11ea-83bb-2f83a5922082.PNG)

**Comment by jedisct1** [comment]
Wow, looks like CHEF-KOCH deleted his GitHub account :(

**Comment by crazy-max** [comment]
@jedisct1 Looks like a ban. All his comments have been removed (no ghost account). Probably related to ghacksuserjs/ghacks-user.js#323?

**Comment by Thorin-Oakenpants** [comment]
> Probably related to ...

It has **NOTHING** to do with the `ghacks userjs` repo (I am the owner) .. I only noticed he was banned because in my weekly github search of "firefox privacy" repos (to see if anything new pops up), his repos were missing in the results

His plagiarism has been noted (we were one of his "victims"), but I have never ever reported him or any of his repos or blogs or other accounts anywhere for anything. I'd rather not waste my time and just ignore him: I banned him from our repos years ago. I have better things to do. He's fixated on me and our user.js - one of the last times he was banned (when one of his gists had links to kiddy porn), he claimed, on reddit, that it was me that reported it, and he threatened to set his lawyers on me (an empty threat as per his standards, he just like to threaten everyone)
